### DISCUSSION TITLE
Extreme Skinsの使い勝手改善等

### DISCUSSION THREAD

**Discussion by yukieiji** [yukieiji/ExtremeRoles#208] [Ideas]
12月の1周年記念アップデートに向けてExtreme Skinsは使い勝手の向上を図るので使いにくいと思った部分を募集します

**Comment by UzomuZo** [comment]
使いにくいとは違う意見もあるかもしれませんが、とりあえず個人的な意見を書いておきます。

１．シェーダーを使用した場合に赤青系統の色が使えない。
２．初めてスキンを作成する際、画像のどの位置にキャラが来るかわからない。（テンプレートがあると良い？）
３．スキンを試作し、動作確認後に微修正した場合、among usの再起動が必要になる。
４．キャラがアニメーションする動作の際に別画像に差し替えられると表現できることが増えるかも。（歩く際は足を上げた画像、はしごの昇り降りの際は左右中央？）

**Reply by yukieiji** [reply]
> １．シェーダーを使用した場合に赤青系統の色が使えない。

AmongUs本体の仕様です

> ２．初めてスキンを作成する際、画像のどの位置にキャラが来るかわからない。（テンプレートがあると良い？）

テンプレートはTORものが使えるのでそれを使って欲しいってのはあります(そのテンプレートについてはライセンスがないのでこちらで勝手に使う改変するってのが出来ないです)

> ３．スキンを試作し、動作確認後に微修正した場合、among usの再起動が必要になる。

画像データを橋渡ししている部分がどうしても起動時に集約してるので再起動はほぼ必須になってしまいます

> ４．キャラがアニメーションする動作の際に別画像に差し替えられると表現できることが増えるかも。（歩く際は足を上げた画像、はしごの昇り降りの際は左右中央？）

AmongUs的にそういうものが想定されてないため実装するとなるとかなり面倒な気がします

**Reply by UzomuZo** [reply]
> AmongUs本体の仕様です
画像データを橋渡ししている部分がどうしても起動時に集約してるので再起動はほぼ必須になってしまいます

うーん、残念。

> テンプレートはTORものが使えるのでそれを使って欲しいってのはあります

それは失礼しました。

> AmongUs的にそういうものが想定されてないため実装するとなるとかなり面倒な気がします

出来たら面白いかもくらいの意見なのでかなり面倒ならいらないですね。

**Comment by yukieiji** [comment]
こちらもアップデート直前になったためディスカッションをロックします
